elements/progress-slider/progress-slider.js

```diff
@@ -123,11 +125,24 @@ let self = class ProgressSlider extends HTMLElement {
 		}
 
 		if (name === "show") {
+			if (newValue === "progress") {
+				this.valueElement.setAttribute("min", 0);
+				this.valueElement.setAttribute("max", 1);
```

Comment:
If `show` is `"progress"` we show a percentage from 0 to 100, not from 0 to 1.

Reply:
Q: Does it mean we need to calculate `progress` differently (for now, it returns a value from 0 to 1), or should we adjust the value accordingly on every change from the spinner (by dividing it by 100)?

Reply:
Good question. I was thinking the latter, but I can see benefits for the former (any math on the value means we need to take steps to avoid silly IEEE754 values like 0.30000000000000004)

Reply:
I'll experiment with it then and make my suggestion in a separate commit. 😀